modules/items/item-ffg.js

```diff
@@ -119,7 +119,7 @@ export class ItemFFG extends ItemBaseFFG {
 
         if (data?.itemattachment) {
           data.itemattachment.forEach((attachment) => {
-            const activeModifiers = attachment.system.itemmodifier.filter((i) => i.system?.active);
+            const activeModifiers = attachment.system.itemmodifier?.filter((i) => i.system?.active);
```

Comment:
Actually this error I was seeing is the result of a migration problem we'll need to tackle.

The attachment object we're looping through here isn't a real instance of an ItemData document, it's an array of our hacky deep embedded items... as such it doesn't have a .system. property if it's from an older item modifier that was added to an item pre-v10.

Reply:
Yeah, I noticed that a few times.